modules/ROOT/pages/configure-apq-cdn.adoc

```diff
@@ -0,0 +1,150 @@
+= Using Automatic Persisted Queries and Content Delivery Networks with Anypoint DataGraph
+
+To improve performance and decrease potential bottlenecks in your network traffic, DataGraph supports automatic persisted queries. You can also use persisted queries in conjunction with content delivery networks (CDNs) to deliver cached results much faster than requiring a roundtrip to the DataGraph server.
+
+== How Automatic Persisted Queries Work in DataGraph
+
+To create a persisted query, you send a POST request that contains:
+* The GraphQL query and any query variables
+* The `persistedQuery` extensions parameter that includes a unique ID, ideally created with a hashing algorithm, such as SHA-256. Using a hash limits the chance of collisions and allows multiple clients to access the query using the same ID. 
+
+After you’ve created the persisted query, you can subsequently send a GET request to DataGraph by passing the hash ID as a persistedQuery extensions parameter.
+
+== Register an Automatic Persisted Query to DataGraph
+
+To post a persisted query ensure you have:
+
+* The DataGraph Consume permission
+* An authenticated connection to DataGraph using a valid client ID and secret
+* Created a ID for the persisted query
+
+To register a persisted query, create a POST request that contains the following:
+
+* The endpoint for the DataGraph instance
+* An extension that contains the `persistedQuery` parameter
+* An ID within the `persistedQuery` parameter
+* A GraphQL query and any variables 
+
+The following example shows how to register a persisted query using cURL:
```

Comment:
I typically see this presented as "curl," all lowercase. I searched and the lowercase seems the most commonly used.